src/NServiceBus.Core/Routing/AutomaticSubscriptions/AutoSubscribe.cs

```diff
@@ -57,6 +57,20 @@ static List<Type> GetMessageTypesHandledByThisEndpoint(MessageHandlerRegistry ha
             return messageTypesHandled;
         }
 
+        class SomeStartupTaskWhichDoesRawSends : FeatureStartupTask {
+            protected override Task OnStart(IMessageSession session)
+            {
+                var rawSession = session.Raw();
```

Comment:
In theory the extension method could also be scoped to FeatureStartupTask only like the following:

static IMessageSessionRaw Raw(this FeatureStartupTask task, IMessageSession session);

usage

```
var rawSession = this.Raw(session);
rawSession.Dispatch()
```
so that it can't be abused on endpoint level for example. But on the other-hand why not allow it?